google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -292,6 +292,18 @@ TEST_F(PgDataTypeIntegrationTest, WriteReadNumeric) {
   EXPECT_THAT(result, IsOkAndHolds(UnorderedElementsAreArray(data)));
 }
 
+TEST_F(PgDataTypeIntegrationTest, WriteReadPgOid) {
+  if (UsingEmulator()) GTEST_SKIP() << "emulator does not support PG.OID";
+
+  std::vector<PgOid> const data = {
+      PgOid("0"),    //
+      PgOid("42"),   //
+      PgOid("999"),  //
+  };
+  auto result = WriteReadData(*client_, data, "PgOidValue");
```

Comment:
This will require updating `PgDatabaseIntegrationTest::SetUpTestSuite()` in `google/cloud/spanner/testing/database_integration_test.cc` to add the "PgOidValue" column.

Reply:
It seems, then, that you'll have to drop this new test case as it depends on having an OID column.

Reply:
Got it, removed.